sampleinput.yaml

```diff
@@ -0,0 +1,20 @@
+- type: github
+  token: my_github_token
+  host: None
```

Comment:
All the `None` values in in this yaml should be `null`. Right now they are being parsed as literal strings.